styleguide/components/00-settings/_colors.scss

```diff
@@ -107,38 +109,24 @@ $tints: (
 $themes: (
   'default': (
     // Required colors!
```

Comment:
Exceptions can be added in the same array(?).
So maybe add a comment // Exceptions too so developers know where to put them.

Perhaps add a little warning too:
// these should be as limited as possible

Reply:
Ok.